src/components/HeaderTodo/HeaderTodo.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { AddNewTodoForm } from '../AddNewTodoForm';
+
+interface Props {
+  user: User | null,
+  title: string,
+  onSetTitle: (newTitle: string) => void,
+  onTodoAdd: (todoData: Omit<Todo, 'id'>) => void;
+  onSetIsError: (newStatus: boolean) => void;
+  onSetErrorText: (newText: string) => void;
+  isAdding: boolean;
+}
+
+export const HeaderTodo: React.FC<Props> = React.memo(({
+  user,
+  title,
+  onSetTitle,
+  onTodoAdd,
+  onSetErrorText,
+  onSetIsError,
+  isAdding,
+}) => {
+  return (
+    <header className="todoapp__header">
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+      <button
+        data-cy="ToggleAllButton"
+        type="button"
+        className="todoapp__toggle-all active"
```

Comment:
this button has 'active' class only when there **all todos are completed**

Reply:
yeap, you`re right, i will make it in the 3rd part of task

Reply:
don't forget to fix it